modules/migration-running-migration-plan-cam.adoc

```diff
@@ -13,13 +13,20 @@ You can stage or migrate applications and data with the migration plan you creat
 The CAM web console must contain the following:
 
 * Source cluster
-* Target cluster, which is added automatically during the CAM tool installation
+* Target cluster
 * Replication repository
 * Valid migration plan
 
 .Procedure
 
-. Log in to the CAM web console on the target cluster.
+. Log in to the source cluster and delete old images:
```

Comment:
Consider breaking this step in half:

```
. Log in to the source cluster as a user with ______ permissions.
. Delete old images by using the following command:
```

Reply:
Permissions step not necessary. Already covered in the entire doc.

Reply:
Then I'll merge it when Travis finishes. Thanks!